backend/db/migrate/20210411020529_add_restaurants_table.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# This migration will add a restaurants table
+class AddRestaurantsTable < ActiveRecord::Migration[6.1]
+  def change
+    create_table :restaurants do |t|
+      t.string :name, null: false
+      t.string :location, null: false
+      t.text :description, null: false
+    end
+
+    add_index :restaurants, :name, unique: true
```

Comment:
Wait, does unique make sense? There are plenty of restaurants that have the same name. Otherwise what are franchise sorts of things supposed to do? `McDonalds #1`, `McDonalds #2`, etc...